tests/suites/test_suite_psa_crypto.function

```diff
@@ -1584,7 +1584,8 @@ void import_export( data_t *data,
     if( PSA_KEY_TYPE_IS_DH( type ) &&
         expected_export_status == PSA_ERROR_BUFFER_TOO_SMALL )
     {
-        export_size -= 8;
+        /* Simulate that buffer is too small, by decreasing it's size by 1 byte. */
```

Comment:
```suggestion
        /* Simulate that buffer is too small, by decreasing its size by 1 byte. */
```
Very minor, not worth pushing on its own.

Reply:
Force-pushed fix.